LayoutKitExampleLayouts/CircleImagePileLayout.swift

```diff
@@ -10,15 +10,17 @@ import UIKit
 import LayoutKit
 
 /// Displays a pile of overlapping circular images.
-public class CircleImagePileLayout: StackLayout {
```

Comment:
might be worth adding an explanatory comment about why we are wrapping StackLayout instead of inheriting.